Artifacts.toml

```diff
@@ -0,0 +1,6 @@
+[aerosol2005]
+git-tree-sha1 = "9da4af348af2606764e5e2c94d9439714221dff1"
+
+    [[aerosol2005.download]]
+    sha256 = "952d723b8462439c266dc0df79d0d4e71f774559edb941e03b7979e8ad743f56"
+    url = "https://caltech.box.com/shared/static/chmel1vdthfvfac0yl61ayw2jqnjzr2c.gz"
```

Comment:
I have to learn more about Artifacts and how do they interact with extensions. Will this file be downloaded always, or only if we are running with aerosols?

Reply:
If you are using the cluster, this file is already on the cluster, so we won't have to download it. 

If you are not using the cluster, this file is downloaded the first time you initialize your package. 

I wrote extensive documentation on artifacts here: https://github.com/CliMA/ClimaArtifacts?tab=readme-ov-file#the-ultimate-guide-to-climaartifacts